build.sbt

```diff
@@ -1,56 +1,63 @@
 name := "IterablePlayUtils"
+organization := "com.iterable"
+homepage := Some(url("https://github.com/Iterable/iterable-play-utils"))
+scmInfo := Some(
+  ScmInfo(url("https://github.com/Iterable/iterable-play-utils"), "scm:git@github.com:Iterable/iterable-play-utils.git")
+)
 
-version := "1.1.1"
+licenses := Seq("Apache-2.0" -> url("http://www.apache.org/licenses/LICENSE-2.0.txt"))
+developers := List(
+  Developer(id = "Iterable", name = "Iterable", email = "engineering@iterable.com", url = url("https://iterable.com"))
```

Comment:
is engineering@ public?

Reply:
actually I'm not sure if it is? I can change it to a different email.